### PR TITLE
Use 'ai_service' parameter to differentiate Prometheus metrics

### PR DESCRIPTION
**Pull request by juanroesel**
Closes ZenHubHQ/devops#2239. 

Also closes ZenHubHQ/devops#2233, as all unit tests are passing locally now, so existing GH Actions should be able to run as expected.

### CODE REVIEW

File: tests/test_llama.py [hunk 1]
```diff
@@ -153,7 +153,9 @@ def mock_kv_cache_seq_add(
 
 def test_llama_patch(mock_llama):
     n_ctx = 128
+    ai_service = "testing"
```
Comment (cwarje) [inline tests/test_llama.py]: Is this intended for the final PR?

File: tests/test_llama.py [hunk 2]
```diff
@@ -230,15 +232,16 @@ def test_utf8(mock_llama):
     llama = llama_cpp.Llama(model_path=MODEL, vocab_only=True, logits_all=True)
 
     output_text = "😀"
+    ai_service = "testing"
```
Comment (cwarje) [inline tests/test_llama.py]: Is this intended for the final PR?

File: tests/test_llama.py (hunk 1 above)
Comment (juanroesel) [reply tests/test_llama.py]: Thanks, Cam. Changing this to a more meaningful value.

**Review by cwarje — APPROVED** [review]
LGTM

**Pull request merged**